test/retry.ts

```diff
@@ -68,7 +70,7 @@ test('retry function gets iteration count', withServer, async (t, server, got) =
 		retry: {
 			calculateDelay: ({attemptCount}) => {
 				t.true(is.number(attemptCount));
-				return attemptCount < 2;
+				return Number(attemptCount < 2);
```

Comment:
Why is this needed? `attemptCount` is not documented to ever be a different value or `undefined`.

Reply:
From reading the source it seems that the return value of `calculateDelay` would be consumed by `computedValue`, which expects a number. When we return `attemptCount < 2`, it is returning a `boolean` type, and to cast it to a number would need `Number()`, or use `attemptCount < 2 ? 1 : 0`

Reply:
`attemptCount < 2 ? 1 : 0` would be much clearer. Can you update https://github.com/sindresorhus/got#testing too?